build/config.props

```diff
@@ -36,8 +36,12 @@
     <VsTargetChannel Condition="'$(IsEscrowMode)' == 'true'">int.d$(VsTargetMajorVersion).$(MinorNuGetVersion)</VsTargetChannel>
     <!-- Specifies the SDK version to download to use for testing. The first value represents the channel, the second value represents the exact SDK version to be download. If a version is not specified, the latest version from the channel will be downloaded.
     Note that multiple SDKs can be downloaded by using `;` as a separator.
+    The channel needs to be two-part version in A.B format. The version needs to be the exact version number if specified.
+    e.g. 5.0;6.0 means install the latest versions from both channel 5.0 and channel 6.0.
+         6.0:6.0.100-preview.7.21379.14 means install the preview version 6.0.100-preview.7.21379.14.
+    Refer to https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script#options for more details.
     -->
-    <CliBranchForTesting Condition="'$(CliBranchForTesting)' == ''">release/5.0.2xx:5.0.200-servicing.21120.4</CliBranchForTesting>
+    <CliBranchForTesting Condition="'$(CliBranchForTesting)' == ''">6.0.4xx</CliBranchForTesting>
```

Comment:
Why not `6.0`?  Wouldn't that pick up 6.0.4xx by default?

For `7.0`, I hope we can just put `7.0` and get the latest branch by default.

Reply:
I just tested, `6.0` is the same with `6.0.3xx`, it will install 6.0.3, not 6.0.4.
```
.\dotnet-install.ps1 -Channel 6.0 -Quality signed -InstallDir .\cli -Version latest -NoPath
dotnet-install: Note that the intended use of this script is for Continuous Integration (CI) scenarios, where:
dotnet-install: - The SDK needs to be installed without user interaction and without admin rights.
dotnet-install: - The SDK installation doesn't need to persist across multiple CI runs.
dotnet-install: To set up a development environment or to run apps, use installers rather than this script. Visit https://dotnet.microsoft.com/download to get the installer.

dotnet-install: Using alternate version 6.0.300 found in https://dotnetstage.blob.core.windows.net/6-0-300-rtm-22213-34-public/Sdk/6.0.300-rtm.22213.34/sdk-productVersion.txt
dotnet-install: Extracting the archive.
dotnet-install: Binaries of dotnet can be found in C:\repos\NuGet.Client\cli\cli\
dotnet-install: Note that the script does not resolve dependencies during installation.
dotnet-install: To check the list of dependencies, go to https://docs.microsoft.com/dotnet/core/install/windows#dependencies
dotnet-install: Installation finished
```